spec/import_result_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe ActiveAdminImport::ImportResult do
+  context "failed_message" do
+    let(:import_result) { ActiveAdminImport::ImportResult.new }
+
+    before do
+      require'pry';binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Put space between the method name and the first argument.<br>Do not use semicolons to terminate expressions.<br>Space missing after semicolon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.